src/components/ButtonGroup/__tests__/ButtonGroup.spec.tsx

```diff
@@ -1,33 +1,44 @@
 import React from 'react';
-import toJson from 'enzyme-to-json';
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
 import { ButtonGroup } from '../ButtonGroup';
 import { Button } from '../../Buttons';
 import { Dropdown } from '../../Dropdown';
 import { ListItem } from '../../List/ListItem';
 
 describe('<ButtonGroup> that renders a button', () => {
     it('should render default button correctly', () => {
-        const wrapper = mount(
+        const view = render(
             <ButtonGroup>
                 <Button key="a">A button</Button>
                 <Button key="b">Another button</Button>
             </ButtonGroup>
         );
-        expect(toJson(wrapper)).toMatchSnapshot();
+        expect(view.asFragment()).toMatchSnapshot();
```

Comment:
I think it is more clean and htmlish if we use `view.container` which returns an HTMLElement to check in the snapshot. What do you think?

Reply:
I don't have a strong preference, and I haven't seen anything in the best practices on this. I agree with you, that it seems more natural to call `view.container` the `view.asFragment()`.  @PavelKirvelTextkernel  what do you think?

Reply:
@eszthoff @andrentk  I think yes we can use it

Reply:
Ok, thanks. I will make a separate PR to change this in the docs and already existing tests.